src/ui/common/src/utils/integrations.ts

```diff
@@ -219,19 +219,32 @@ export async function connectIntegration(
       config[k] = '';
     }
   });
-  const res = await fetch(`${apiAddress}/api/integration/connect`, {
-    method: 'POST',
-    headers: {
-      'api-key': user.apiKey,
-      'integration-name': name,
-      'integration-service': service,
-      'integration-config': JSON.stringify(config),
-    },
-  });
 
-  if (!res.ok) {
-    const message = await res.json();
-    throw new Error(message.error);
+  try {
+    const res = await fetch(`${apiAddress}/api/integration/connect`, {
+      method: 'POST',
+      headers: {
+        'api-key': user.apiKey,
+        'integration-name': name,
+        'integration-service': service,
+        'integration-config': JSON.stringify(config),
+      },
+    });
+
+    if (!res.ok) {
```

Comment:
Just making sure I understand the API's response correctly here, when an integration connects successfully do we just return a 200 OK and no response body?

Reply:
Yep!